scm/driver/bitbucket/content.go

```diff
@@ -57,6 +65,13 @@ type content struct {
 	Attributes []string `json:"attributes"`
 }
 
+type metaContent struct {
+	Path   string `json:"path"`
+	Commit *struct {
```

Comment:
the pointer implies the value can be null, but at line 33 we do not do a null check.  Do we need to worry about a null pointer here?  I would recommend we remove the pointer mostly for the purpose of consistency with the rest of the code.

Reply:
FYI this is a really useful tool for generating structs from JSON samples https://mholt.github.io/json-to-go/.  Less useful for smaller JSON objects like this one, but very useful for larger objects.